.travis.yml

```diff
@@ -41,4 +41,5 @@ script: |
       cargo fmt -- --check && cargo clippy
   elif [[ $TASK = "test" ]]; then
       cargo build && cargo run -- prepare-local --docker-env mini && cargo test
+      ./tests/check-config/test_check_config.sh
```

Comment:
You should concatenate this to the first line (with `&&`), otherwise Travis won't report the correct exit status.

Reply:
That's fixed now.